src/translate/compute.rs

```diff
@@ -26,16 +26,17 @@ pub fn translate_computation(meta: &TranslateMetadata, operation: ArithOp, left:
     match meta.arith_module {
         ArithType::BcSed => {
             let (left, right) = (left.unwrap_or_default(), right.unwrap_or_default());
-            let mut math_lib_flag = true;
-            // Removes trailing zeros from the expression
-            let sed_regex = "/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//";
```

Comment:
I agree that this should be fixed... but I think that we need to somehow implement this under the hood for `absurd_cast` for `Text` to `Num`:

```ab
#[allow_absurd_cast]
fun parse(text: Text) {
	return text as Num
}
```

Since `text` variable could be for instance `"0000001"` which should evaluate to `1`

Reply:
maybe is better `to_string` as method but:

```
echo "scale=0; 000001" | bc -l
```

Return just `1`.